integration_tests/CMakeLists.txt

```
@@ -436,7 +436,8 @@ RUN(NAME complex_05 LABELS gfortran llvm)
 RUN(NAME complex_06 LABELS gfortran llvm)
 RUN(NAME complex_07 LABELS gfortran) # cmplx
 RUN(NAME complex_08 LABELS gfortran) # conjg
-RUN(NAME complex_09 LABELS gfortran) # conjg
+RUN(NAME complex_09 LABELS gfortran)
+RUN(NAME complex_10 LABELS gfortran) # conjg
```

If this works with LFortran, then enable LFortran:
```suggestion
RUN(NAME complex_10 LABELS gfortran llvm) # conjg
```